manim/mobject/svg/text_mobject.py

```diff
@@ -476,6 +476,8 @@ def __init__(
         self.text = text
         if self.disable_ligatures:
             self.submobjects = [*self.gen_chars()]
+            if config.renderer == "opengl":
+                self.assemble_family()
```

Comment:
I looked through `OpenGLMobject`'s methods and it has a `.set_submobjects()` method which calls `assemble_family()` through `.add` and `.remove`. It's probably a better fit for this
```py
    def set_submobjects(self, submobject_list):
        self.remove(*self.submobjects)
        self.add(*submobject_list)
        return self
```
```suggestion
            if config.renderer == "opengl":
                self.set_submobjects(self.gen_chars())
            else:
                self.submobjects = [*self.gen_chars()]
```